pytorch_tcn/tcn.py

```diff
@@ -547,9 +420,9 @@ def __init__(
             embedding_shapes: Optional[ ArrayLike ] = None,
             embedding_mode: str = 'add',
             use_gate: bool = False,
-            lookahead: int = 0,
```

Comment:
ordering of parameters should stay the same

Reply:
Fixed.